erpnext/vehicles/report/vehicle_maintenance_schedule/vehicle_maintenance_schedule.js

```diff
@@ -90,5 +90,41 @@ frappe.query_reports["Vehicle Maintenance Schedule"] = {
 			fieldtype: "Link",
 			options: "Item Group"
 		},
-	]
-};
\ No newline at end of file
+	],
+
+	onChange: function(new_value, column, data, rowIndex) {
+		if (column.fieldname == "remarks") {
+			if (cstr(data['remarks']) === cstr(new_value)) {
+				return
+			}
+
+			if (!data.opportunity) {
+				frappe.msgprint(__("Opportunity does not exist"));
```

Comment:
Just do `frappe.throw` .. no need to reload I think .. Datatable will handle the exception and revert the old value

Reply:
frappe.throw is not reverting the cell to old value